src/biogeophys/Wateratm2lndBulkType.F90

```diff
@@ -324,6 +337,9 @@ subroutine InitAccVars(this, bounds)
        call extract_accum_field ('PREC60', rbufslp, nstep)
```

Comment:
This is something not to worry about now. But, seeing this it looks like we have various running mean lengths of fields that are technically only used by one of the Li fire methods. But, we always do all of them no matter what option is selected.

That's probably OK, as it's not going to matter too much. But, it might mean that eventually we'll want to remove the oldest fire option, just so that we can remove the accum fields that aren't used by newer ones. Or it might mean we have a way of flagging which accum fields are actually needed, and only do those ones instead of everything.

This is longer term code health that I'm mentioning because I saw it here. But, not something we'll worry about doing anything about until some time in the future...

Reply:
Added this to my new issue #2714.